### DISCUSSION TITLE
chatgpt plus / gpt-4 api server

### DISCUSSION THREAD

**Discussion by hoonlight** [waylaidwanderer/node-chatgpt-api#303] [Help] [answered]
Hello, I am trying to use gpt-4 by paying for chatgpt plus. But can I use this through node-chatgpt-api server? (I'm using bing chat to discord bot through node-chatgpt-api server).

thank you

**Comment by waylaidwanderer** [comment, marked as answer]
You can use `ChatGPTBrowserClient` to do so.

**Reply by hoonlight** [reply]
thank you, I'll try

**Reply by hoonlight** [reply]
> You can use `ChatGPTBrowserClient` to do so.

But how can I select a model in chatGptBrowserClient? I have to choose between gpt-3.5 and gpt-4 models. chatGptBrowserClient in settings.js has no option to set model.

**Reply by allentofight** [reply]
+1, I have the same question, how to choose gpt-4 model

**Reply by hoonlight** [reply]
> +1, I have the same question, how to choose gpt-4 model

https://github.com/acheong08/ChatGPT

Unfortunately I couldn't solve this problem, so I chose to use another library. now it works fine

**Reply by waylaidwanderer** [reply]
@comienzo2093 @allentofight As part of ChatGPTBrowserClient's options, you can set the `model` property which is `text-davinci-002-render-sha` by default. I don't remember what the GPT-4 model is called but you can check using devtools and looking at the `models` endpoint request on chat.openai.com.

https://github.com/waylaidwanderer/node-chatgpt-api/blob/main/src/ChatGPTBrowserClient.js#L29
